Neo4j.Driver/Neo4j.Driver/Internal/Protocol/BoltProtocolV4_0.cs

```diff
@@ -154,7 +154,34 @@ await connection
         protected internal override void GetProcedureAndParameters(IConnection connection, string database, out string procedure, out Dictionary<string, object> parameters)
         {
             procedure = GetRoutingTableForDatabaseProcedure;
-            parameters = new Dictionary<string, object> { { "context", connection.RoutingContext }, { "database", string.IsNullOrEmpty(database) ? null : database } };                     
+            parameters = new Dictionary<string, object> { { "context", connection.RoutingContext }, { "database", string.IsNullOrEmpty(database) ? null : database } };
+        }
+
+        public override async Task<IReadOnlyDictionary<string, object>> GetRoutingTable(IConnection connection, string database, string impersonatedUser, Bookmark bookmark)
```

Comment:
What was the reason to override this? I can't see any obvious difference from the parent class implementation.

Reply:
in v3 we pass null instead of bookmark in RunInAutoCommitTransactionAsync